config/jobs/etcd/etcd-presubmits.yaml

```
@@ -822,6 +822,38 @@ presubmits:
       nodeSelector:
         kubernetes.io/arch: arm64
 
+  - name: pull-etcd-functional-amd64
+    cluster: k8s-infra-prow-build
+    always_run: true
+    branches:
+      - release-3.4
+    decorate: true
+    decoration_config:
+      timeout: 60m
+    annotations:
+      testgrid-dashboards: sig-etcd-presubmits
+      testgrid-tab-name: pull-etcd-functional-amd64
+    spec:
+      containers:
+      - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250513-98d205aae3-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - |
+            apt-get update && apt-get install -y netcat-openbsd
+            GOARCH=amd64 make test-functional
```

We can also remove `GOARCH` here, as it will run on amd64.

```suggestion
            make test-functional
```

---

I would tend to leave this in to keep the invocation appearances similar across all the jobs. Happy either way.

---

Let's leave it then :)